src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3444,7 +3455,10 @@ void SimAirLoopComponents(EnergyPlusData &state,
                                 FirstHVACIteration,
                                 AirLoopNum,
                                 PrimaryAirSystems(AirLoopNum).Branch(BranchNum).Comp(CompNum).CompIndex,
-                                PrimaryAirSystems(AirLoopNum).Branch(BranchNum).Comp(CompNum).compPointer);
+                                PrimaryAirSystems(AirLoopNum).Branch(BranchNum).Comp(CompNum).compPointer,
+                                AirLoopNum,
+                                BranchNum,
+                                CompNum);
         } // End of component loop
```

Comment:
Allows calling point for "CoilSystem:Cooling:Water" placed in airloop. Also adds three new arguments to SimAirLoopComponent() function that allows getting the pointer to the component simulated if it is not already set. This approach is adopted from CoilSystem-to-Unitary branch.